lib/init.coffee

```diff
@@ -1,8 +1,56 @@
+{CompositeDisposable} = require 'atom'
+helpers = require 'atom-linter'
+XRegExp = require('xregexp').XRegExp
+
 module.exports =
   config:
-    xmllintExecutablePath:
+    executablePath:
       type: 'string'
-      default: ''
-
+      title: 'xmllint Executable Path'
+      default: 'xmllint'
   activate: ->
-    console.log 'activate linter-xmllint'
+    require('atom-package-deps').install('linter-xmllint')
```

Comment:
Since this is a new package you should depend on version 3 and just leave the parameter for `install()` empty.